### PR TITLE
added OpenGTS to Maps & GPS section

### PR DESCRIPTION
**Pull request by troyanov**
OpenGTS™ ("Open GPS Tracking System") is the first available open source project designed specifically to provide web-based GPS tracking services for a "fleet" of vehicles

### CODE REVIEW

**Comment by nodiscc** [comment]
Please remove "Written in Java and can be easily extended for your needs " and add the language tag `Java`

File: README.md [hunk 1]
```diff
@@ -587,6 +587,7 @@ Some [Groupware](#groupware) solutions also feature file sharing and synchroniza
...
   * [MapBBCodeShare](https://github.com/MapBBCode/share.mapbbcode.org) - Tool for sharing custom OSM maps. Support for annotated markers, polygons, lines, multi-format import/export, multiple layers, shortlinks... ([Demo](http://share.mapbbcode.org/)) `WTFPL/Other` `PHP/Javascript`
   * [Openstreetmap](http://www.openstreetmap.org/) - OpenStreetMap is a map of the world, created by people like you and free to use under an open license. ([Source Code](https://github.com/openstreetmap/openstreetmap-website))
+  * [OpenGTS](http://www.opengts.org/) - Was designed to fill the needs of an entry-level fleet tracking system, it is also very highly configurable and scalable to larger enterprises as well. Supports variety of tracking devices and protocols. Has a web interface to view your trips and for freel-management. Provides a variety of customization options as well as reporting feature. Written in Java and can be easily extended for your needs ([Demo](http://track.opengts.org/track/Track), [Source Code](https://sourceforge.net/projects/opengts/files/server-base/)) `Apache`
```
Comment (Kickball) [inline README.md]: Please shorten description to 250 characters and end it with a full stop.

Comment (Kickball) [inline README.md]: Please add a language tag after the Apache tag, for the programming language the project uses (Java).

[Source](http://www.opengts.org/FAQ.html#faq_install)

Comment (Kickball) [inline README.md]: Please order alphabetically (between MapBBCodeShare and OpenStreetMap).

**Pull request merged**